### ISSUE TITLE
Context Selector - aria refinements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1531]
A couple of refinements can be made to the context selector:
1. include a reference to the hidden span from the toggle
2. use the label on the search button as the label for the input

**Details:**

1. `pf-c-context-selector__toggle` should have an `id` and an `aria-labelledby` that references the `id`s of the hidden span and `pf-c-context-selector__toggle`.
2. The search input can be labelled by the button label, as shown below. Note that I removed the `id` and `name` attributes from the `<input>`. I'm not sure that those are needed or not.
```
<input class="pf-c-form-control" type="search" placeholder="Search" aria-labelledby="search-button">
<button class="pf-c-button pf-m-tertiary" id="search-button" aria-label="Search menu items">
  <i class="fas fa-search" aria-hidden="true"></i>
</button>
```
